portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,26 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Initializes the dropdown menu on the top fixed navbar.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+$(document).ready(function(){
+    $(".dropdown-trigger").dropdown();
+});
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+
+
+/**
+ * Adds a random fun fact about me on each click.
+ */
+
+function randomFunFacts() {
+  const funFacts =
+      ['I"m a  Rising sophomore Computer Science major', 'This is my first internship', 'I was raised and high-schooled in Lagos, Nigeria', 'I speak three languages: English, and two native lanuages, Yoruba and Efik'];
```

Comment:
This line is too long. Normally we recommend keeping each line short when possible for better readability. Can this array of strings split into different lines?

Reply:
Okay, cool. I will do that. After making the change do I need to make a commit and send a new PR ?

Reply:
You don't need to send a new PR. Just create new commit as you have been doing so far.

Reply:
Okay, will do that. And for the screenshot, I can't find a way to include images in this comment box.

Reply:
Does this help? https://help.github.com/en/github/managing-your-work-on-github/file-attachments-on-issues-and-pull-requests

Reply:
Yes, it does. This is the current stage I am.
![Screenshot 2020-05-27 at 3 54 38 PM](https://user-images.githubusercontent.com/39606715/83066101-88abf080-a032-11ea-8cfc-3c94cea9ae83.png)